docs/general/events.md

```diff
@@ -98,6 +98,15 @@ Recommendations on using attributes vs. body fields:
   * The _fields_ are unique to the named event (`event.name`) and different events
     may use the same _field_ name to represent different data, due to the unique
     nature of the event.
+  * The _fields_ SHOULD NOT use the `*.blob_ref.*` name, except when applying the
```

Comment:
NIt:  You can provide a policy to enforce this in the `/policies` directory.  Would like to see this happen after this naming and feature is approved.

Reply:
Sounds good. I'm happy to either add this enforcement in a separate PR or to add to the existing PR. Would it be better to do that separately or in a single PR?

Reply:
I've since added `*.rego` policy files for this. OK to resolve?